src/renderer/src/routes/pages/Neo3NeoXBridge/Neo3NeoXBridgeContent.tsx

```diff
@@ -260,9 +262,16 @@ export const Neo3NeoXBridgeContent = ({ account }: TProps) => {
   }
 
   const handleSubmit = async () => {
-    if (!isBridgeValid) return
+    if (!isBridgeValid || !actionData.accountToUse.value) return
```

Comment:
Not necessary, just:
```suggestion
    if (!isBridgeValid) return
```